### ISSUE TITLE
libc16 for Ubuntu 20.04

### ISSUE THREAD

**Issue by scorpioza** [ligfx/gimp-creatures-sprites#1]
Hi! I cannot install plugin because of libc16. Recently I installed **elks-libs** but when I run ./configure I see the same error:

```
configure: error: Package requirements (c16) were not met:
No package 'c16' found
```

Thanks in advance! I

**Comment by ligfx** [comment]
Hi @scorpioza , libc16 can be found on Sourceforge: https://sourceforge.net/projects/c3-ds-tools/files/libc16/0.3/

**Comment by scorpioza** [comment]
ligfx, thank you for your help! 
I installed libc16 and libc16-dev, but result is the same: **No package 'c16' found**

```
$ whereis libc16
libc16: /usr/lib/libc16.a /usr/lib/libc16.so
```

I suppose that libc16 is for i386 and I am on x86_64, so ubuntu doesn't find this dependency. I had the similar problem (not the same) here: https://creaturescaves.com/forum.php?view=3&thread=6603 (last post) and I found solution in askubuntu. I will try to solve this problem and if I succeed, I will write a solution.
Have a nice day!
